ClientLib/src/main/java/com/o3dr/android/client/Drone.java

```diff
@@ -530,6 +530,10 @@ public void triggerCamera() {
         ExperimentalApi.triggerCamera(this);
     }
 
+    public void setROI(double latitude, double longitude, double altitude) {
```

Comment:
@Teemperor I'm moving away from exposing functionality in this manner, otherwise this class will grow very large very fast as features keep being added.

The new model to follow is to make use of the specific api interface (i.e: ExperimentalApi, GuidedApi), so in this case just adding the `setROI` method to the `ExperimentalApi` class should be sufficient.

A future update will deprecate the use of the `Drone` class methods that are provided by a specific api interface.